AppController/djinn.rb

```diff
@@ -3746,6 +3746,13 @@ def start_hermes()
       @options["write_detailed_processes_stats_log"].downcase == 'true',
       @options["write_detailed_proxies_stats_log"].downcase == 'true'
     )
+    if my_node.is_shadow?
+      nginx_port = 17441
+      service_port = 4378
+      Nginx.add_service_location(
+        'appscale-administration', my_node.private_ip,
+        service_port, nginx_port, '/stats/cluster/')
```

Comment:
`"appscale-administration"`is a service name. This service has two locations:
`"/stats/cluster"` which is handled by Hermes and `"/"` (other URL paths) which is handled by Admin Server.
So locations are different, but they works over the same port and they are parts of the same service.